test/giveaway_test.go

```diff
@@ -34,7 +34,7 @@ func (s *GiveawayTestSuite) TestDirectTransfer() {
 		}
 		time.Sleep(15 * time.Second)
 	}
-	s.verifyBitmark(bitmarkID, s.receiver.AccountNumber(), "transferring", 0)
+	s.verifyBitmark(bitmarkID, s.receiver.AccountNumber(), "transferring", 5*time.Second)
```

Comment:
Allocate some waiting time for updaterd to update the status; otherwise sometimes the test case fails.

Reply:
This fix is not related to this PR.